simpleprophet/simpleprophet/models.py

```diff
@@ -202,7 +196,11 @@ def data_filter(data, product):
         "Firefox iOS Global MAU": [s2d('2017-11-08'), s2d('2017-12-31')],
         "Firefox iOS Tier1 MAU": [s2d('2017-11-08'), s2d('2017-12-31')],
         "Mobile Global MAU": [s2d('2017-11-10'), s2d('2018-03-11')],
+        "Mobile Global MAU": [s2d('2019-12-04'), s2d('2020-01-27')],
+        "Mobile Global MAU": [s2d('2020-08-01'), s2d('2020-10-08')],
         "Mobile Tier1 MAU": [s2d('2017-11-10'), s2d('2018-03-11')],
+        "Mobile Tier1 MAU": [s2d('2019-12-04'), s2d('2020-01-27')],
+        "Mobile Tier1 MAU": [s2d('2020-08-01'), s2d('2020-10-08')],
```

Comment:
`anomalyDates` is a `dict` these multiple entries with the same key `"Mobile Tier1 MAU"` are going to overwrite one another.  It looks like this code as currently implemented doesn't support more than one anomaly date range per product.

Reply:
I have updated the structure, and the function to take multiple anomaly date range per product